static/js/trips-map.js

```diff
@@ -29,19 +34,50 @@ map.fitBounds([[
 ], [
     Math.max.apply(null, latitudes) + 1,
     Math.max.apply(null, longitudes) + 1
-]]);
+]])
 
 // Current location
 if(current_location.length) {
   const popup = new mapboxgl.Popup({offset: [10, -20]})
       .setText('Latest location')
 
-  const el = document.createElement('div');
+  const el = document.createElement('div')
   el.classList.add('marker')
   el.classList.add('current')
 
+  lineCoords.unshift(current_location)
+
   new mapboxgl.Marker(el, {offset:[0, -30]})
       .setLngLat(current_location.reverse())
       .setPopup(popup)
       .addTo(map)
 }
+
+// Create lines
+map.on('load', _ => {
+    map.addLayer({
+        'id': 'route',
+        'type': 'line',
+        'source': {
+            'type': 'geojson',
+            'data': {
+                'type': 'Feature',
+                'properties': {},
+                'geometry': {
+                    'type': 'LineString',
+                    'coordinates': lineCoords
+                }
+            }
+        },
+        'layout': {
+            'line-join': 'round',
+            'line-cap': 'round'
+        },
+        'paint': {
+            'line-color': 'rgba(0, 0, 255, 0.5)',
```

Comment:
Suggestion for a color and transparency: `rgba(212, 85, 85, 0.3)`.
I tried a few things and this seems to me like a good compromise.